pontoon/contributors/static/js/profile.js

```diff
@@ -82,68 +84,79 @@ var Pontoon = (function (my) {
-                label: function (items, chart) {
-                  const label = chart.datasets[items.datasetIndex].label;
-                  const value = nf.format(items.yLabel / 100);
-
-                  return `${label}: ${value}`;
+            clip: false,
+            plugins: {
+              tooltip: {
+                mode: 'index',
+                intersect: false,
+                borderColor: style.getPropertyValue('--status-translated'),
+                borderWidth: 1,
+                caretPadding: 5,
+                padding: {
+                  x: 10,
+                  y: 10,
+                },
+                callbacks: {
+                  labelColor: this.setLabelValue,
+                  label: function (context) {
+                    const { parsed, chart, datasetIndex } = context;
+                    const label = chart.data.datasets[datasetIndex].label;
+                    const value = nf.format(parsed.y / 100);
+
+                    return `${label}: ${value}`;
+                  },
                 },
               },
             },
             scales: {
-              xAxes: [
-                {
-                  type: 'time',
-                  time: {
-                    displayFormats: {
-                      month: 'MMM',
-                    },
-                    tooltipFormat: 'MMMM YYYY',
-                  },
-                  gridLines: {
-                    display: false,
-                  },
-                  offset: true,
-                  ticks: {
-                    source: 'data',
+              x: {
+                type: 'time',
+                time: {
+                  displayFormats: {
+                    month: 'MMM',
                   },
+                  tooltipFormat: 'MMMM yyyy',
                 },
-              ],
-              yAxes: [
-                {
-                  gridLines: {
-                    display: false,
-                  },
-                  position: 'right',
-                  ticks: {
-                    beginAtZero: true,
-                    maxTicksLimit: 3,
-                    max: 100,
-                    precision: 0,
-                    callback: (value) => nf.format(value / 100),
-                  },
+                grid: {
+                  display: false,
+                },
+                offset: true,
+                ticks: {
+                  source: 'data',
+                },
+              },
+              y: {
+                grid: {
+                  display: false,
                 },
-              ],
+                position: 'right',
+                ticks: {
+                  maxTicksLimit: 3,
+                  precision: 0,
+                  callback: (value) => nf.format(value / 100),
+                },
+                beginAtZero: true,
+                max: 100,
+              },
             },
           },
         });
       },
+      setLabelValue: function (context) {
```

Comment:
We can simplify this further I believe by only having one `setLabelValue()` located in `insights_charts.js`, similar to how `customLegend()` was used.

Also, is `setLabelColor()` a better name?